src/main/java/webApp/ServingWebContentApplication.java

```diff
@@ -3,12 +3,12 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.ComponentScan;
 
-@SpringBootApplication(scanBasePackages={"repository"})
-@EntityScan(basePackages = {"repository"})
+@SpringBootApplication
+@EntityScan(basePackages = {"survey"})
```

Comment:
Can we have the word "survey" as a constant for later use in this class

Reply:
"survey" isn't used later in the class.
I don't know what the advantage of having it as a constant is. This name must correspond to a package name. Making it a constant means the reviewer has to look elsewhere to make sure that the value is correct, so doesn't save any time.

Constants make sense in situation where the value matters little but it is used in many places. For example, the ring buffer on the arduino has a macro that sets the size of the buffer and is used in the read and write functions to validate the length of the buffer to prevent overflow. There, the macro constant is used many places, but the actual value could be anything over 1, so it is easy to change. In this case, "survey" doesn't seem to be used in other places, and is also integral to the code working here so it's ability to change is limited. I suppose it might change if we rename the other package, but in that case refactoring is required anyway so unless we have it everywhere I don't know the advantage.

Maybe there is a config file where this would be more appropriate so it applies everywhere, but I don't know what that would be.

Reply:
So this is what referred to maybe having in the "localhost:8080" comment I had about the potential for having a properties or constant holder file. I don't know if this that would even be applicable here but we can discuss it in the future.